reconcile/test/fixtures/helm/aws_account_shard_disabled.yml

```diff
@@ -47,8 +47,68 @@ objects:
                 key: log_group_name
           command: ["/bin/sh", "-c"]
           args:
+          # generate /etc/vector/vector.yaml
```

Comment:
It may be possible to come up with a way to auto-generate all 28 test fixtures, but that is very likely to be out of scope for this spike.

Reply:
This is just about your speed and efficiency. If I was doing this, I would already have ChatGPT write me a script to do this automatically so that I don't waste time on it.